packages/observe-sequence/observe_sequence.js

```diff
@@ -126,7 +126,7 @@ ObserveSequence = {
   fetch: function (seq) {
     if (!seq) {
       return [];
-    } else if (_.isArray(seq)) {
+    } else if (seq instanceof Array || _.isArray(seq)) {
```

Comment:
Can you declare a helper function in this file called `isArray` that has this implementation?